### PR TITLE
added size method to coffee-resque to get number of jobs in queue.

### PR DESCRIPTION
**Pull request by MishaConway**
Minor addition that allows you to asynchronously get the size of a queue. If you think this a good idea, but have any questions or concerns feel free to let me know. 

Example usage is:

resque.size( 'emails', function (err, count) {
    console.log( "number of jobs in emails is: " + count );
})

### CODE REVIEW

**Comment by MishaConway** [comment]
@steelThread I have needed to keep using my fork of coffee-resque since I have some cases where I need to get the size of a queue. Is this something that can be considered to be merged in? No worries if you think this is outside the scope of the project.

**Comment by steelThread** [comment]
Sorry, I just haven't gotten around to looking at it. I'll peek at your pr when I get home tonight.

File: src/index.coffee [hunk 1]
```diff
@@ -48,6 +48,21 @@ class Connection
...
+  #
+  # queue     - String queue name.
+  # callback  - Callback with signature (err, response) to run when size is retrieved.
+  #             Err contains the error if there was an issue retrieving the size. If
+  #             there is no error, err will be null and response will contain the size
+  #             as an integer.
+  #
+  # Returns nothing.
+  size: (queue, callback) ->
+    @redis.llen this.key('queue', queue), callback
```
Comment (steelThread) [inline src/index.coffee]: Can you use the coffeescript shorthand for `this` here please, ie `@key`?

**Comment by steelThread** [comment]
Ruby resque supports a similar [size](https://github.com/resque/resque/blob/master/lib/resque.rb#L206) utility so this is a fine addition.  I'll happily merge this PR after my comment above is addressed.

File: src/index.coffee (hunk 1 above)
Comment (MishaConway) [inline src/index.coffee]: @steelThread Good catch. I've made the update.

**Pull request merged**